crates/bevy_animation/src/lib.rs

```diff
@@ -416,6 +473,21 @@ fn run_animation_player(
             parents,
             children,
         );
+        if animation.root_motion.is_some() {
+            delta = delta.lerp(transition_delta, *current_weight);
+        }
+    }
+
+    if delta != Vec3::ZERO {
+        // SAFETY: The verify_no_ancestor_player check above ensures that two animation players cannot alias
```

Comment:
I don't see any check happening above. Are you referring to the actual function or a call that happens in `apply_animation`? (It appears this line was copied)